# HG changeset patch
# User Richard Russon <rich@flatcap.org>
# Date 1464576248 -3600
#      Mon May 30 03:44:08 2016 +0100
# Branch gentoo-1.7
# Node ID e4765fbdfd9672ae99f9081f8dd0c7ed000c08cf
# Parent  64ee94b76133a6e434f1c171f9e0165c7305d19c
[PATCH 05/11] xterm-set-only-window-title
From b79628774cd1cb5279592233cba3889870d6028f Mon Sep 17 00:00:00 2001
user:        Fabian Groffen <grobian@gentoo.org>
date:        Sun Oct 27 16:29:18 2013 +0100
files:       curs_main.c
description:
fix xterm-titles patch to not set both window and icon title
---
 curs_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -r 64ee94b76133 -r e4765fbdfd96 curs_main.c
--- a/curs_main.c	Mon May 30 03:42:56 2016 +0100
+++ b/curs_main.c	Mon May 30 03:44:08 2016 +0100
@@ -129,7 +129,7 @@ static const char *No_visible = N_("No v
    (option (OPTCOLLAPSEFLAGGED) || FLAGGED(header) == 0))
 
 /* de facto standard escapes for tsl/fsl */
-static char *tsl = "\033]0;";
+static char *tsl = "\033]2;";
 static char *fsl = "\007";
 
 /* terminal status capability check. terminfo must have been initialized. */
